databases/week2/week2.sql

```diff
@@ -0,0 +1,97 @@
+-- Active: 1738674433543@@127.0.0.1@3306@my_database_week1
+-- part 1
+SELECT * from status
+SELECT * FROM task;
+
+INSERT into task(title, description, created, updated, due_date, status_id, user_id)
+VALUES('Database week2','Do homework for database week 2', '2025-02-09 12:00:00', '2025-02-13 17:10:15','2025-02-12 18:00:00',3,1);
+
+UPDATE task
+SET title = "HUF homework"
+WHERE title= "Database week2";
```

Comment:
Nice and simple statements; excellent! ⭐

As a small note, I'd be wary of updating on anything but a unique identifier as you might update more data than you expect or less for that matter. I can see you're switching to IDs below, so I am probably preaching to the choir. 😄